### DISCUSSION TITLE
API call limiter

### DISCUSSION THREAD

**Discussion by mizaki** [comictagger/comictagger#463] [Ideas]
What do you think about adding [Pyrate-Limter](https://github.com/vutran1710/PyrateLimiter) to CT. For external talkers they could choose to use what they wanted but it might be better to put one in the main project? The limit code for CV could be improved with it too.

**Comment by lordwelch** [comment]
I don't see a use to adding it to ComicTagger proper (comicatggerlib) but using it in ComicVineTalker would let us be compliant with ComicVine's rules.
